app/lib/verify-request.server.ts

```diff
@@ -36,7 +36,7 @@ export const verifyRequest = <UserType extends "customer" | "employee">(
     const decodedJwt = verifyJwt(jwt);
     let parsedData: z.SafeParseReturnType<any, any>;
     const schema = z.object({
-      id: z.string(),
+      id: z.string().cuid(),
```

Comment:
Not strictly necessary, because we could change the `cuid` with a simple `uuid` and Im not sure if this will still work.